src/elan-cli/elan_mode.rs

```diff
@@ -344,13 +320,8 @@ fn show(cfg: &Cfg) -> Result<()> {
         if show_headers {
             print_header("installed toolchains")
         }
-        let default_name = cfg.get_default()?;
         for t in installed_toolchains {
-            if default_name.as_ref() == Some(&t) {
-                println!("{} (default)", t);
-            } else {
-                println!("{}", t);
-            }
+            println!("{}", t);
```

Comment:
Why doesn't it show the default anymore? This was a useful functionality and one of the reasons I use `elan show`.

Reply:
To me the main use of `show` is to document the choice of active toolchain, where `default` is still displayed. Do you have other use cases in mind? Should `elan default` without argument show the (unresolved) option value?

Reply:
It shows the default again